src/autoyast-rnc/auth-client.rnc

```diff
@@ -1,301 +0,0 @@
-default namespace = "http://www.suse.com/1.0/yast2ns"
```

Comment:
Last year I learned that .rnc file is only used for validating AutoYast XML input, it plays no role in the serialisation/deserialisation of Import/Export interface data, hence the file is removed because:
- It defines elements that clearly violate XML, such as those beginning with letter 'l' and 'm'.
- There are way too many configuration keys that would be painful to be serialised into XML and hence the module exports/imports JSON instead.